### PR TITLE
Use SQLite as storage

### PR DESCRIPTION
**Pull request by tautropfli**

### CODE REVIEW

File: hirzenbach-bot/commands.py [hunk 1]
```diff
@@ -114,21 +102,24 @@ async def inspire(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
...
     )
+    chat_memory = persistence.get_memory(update.effective_chat.id)
 
     reply_to_message = update.effective_message.reply_to_message
     is_reply_to_me = (
         reply_to_message is not None and reply_to_message.from_user.id == context.bot.id
     )
     is_addressing_me = context.bot.name in update.effective_message.text
 
+    fraction_match = _FRACTION_REGEX.match(update.effective_message.text)
```
Comment (janhohenheim) [inline hirzenbach-bot/commands.py]: You can register a handler directly for this regex in `main.py`

File: hirzenbach-bot/main.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 from threading import Thread
 import gpt3
 from env import Env
-from persistance import Data
+from persistence import init_database
```
Comment (janhohenheim) [inline hirzenbach-bot/main.py:15]: can't we delete `persistance.py` now? Apologies if you already did and I misinterpreted the UI.

File: hirzenbach-bot/vulgar_fraction.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    return ''.join((format_digit(digit) for digit in _digits(n)))
+
+
+def _digits(n: int) -> Iterable[str]:
+    return str(n)
+
+
+def _superscript(character: str) -> str:
+    match character:
+        case '-':
```
Comment (janhohenheim) [inline hirzenbach-bot/vulgar_fraction.py:36]: no `+`?

**Review by janhohenheim — CHANGES_REQUESTED** [review]

File: hirzenbach-bot/vulgar_fraction.py (hunk 3 above)
Comment (tautropfli) [reply hirzenbach-bot/vulgar_fraction.py:36]: We never format integers with explicit leading +, so not necessary

File: hirzenbach-bot/main.py (hunk 2 above)
Comment (tautropfli) [reply hirzenbach-bot/main.py:15]: Persist**a**nce got renamed to Persist**e**nce (the diff shows it as renamed to migrate.py)

File: hirzenbach-bot/commands.py (hunk 1 above)
Comment (tautropfli) [reply hirzenbach-bot/commands.py]: Uu that's really cool! [fc2a085](https://github.com/janhohenheim/hirzenbach-bot/pull/1/commits/fc2a085ca1ff15676d9e7502e4c5f6355a930765)

**Pull request merged**